worker.js

```diff
@@ -0,0 +1,56 @@
+var Sway = require('sway')
```

Comment:
oh if you add karma test runner and two simple test (happy path and errors) it would be golden!